app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+
+  def check_last_three_posts
+    posts.limit(3).order(created_at: :desc)
```

Comment:
[REQUIRED] Kindly change this to: 
`  posts.order('created_at Desc').limit(3)`
Your code will get the first three posts and then arrange them according to descending order instead of arranging in descending then getting the first 3